src/main/java/io/cryostat/targets/Target.java

```diff
@@ -290,7 +293,17 @@ void prePersist(Target target) {
             if (!Objects.equals(encodedAlias, target.alias)) {
                 target.alias = encodedAlias;
             }
-
+            try {
+                if (!uriUtil.validateUri(target.connectUrl)) {
+                    throw new IllegalArgumentException(
+                            String.format(
+                                    "Connect URL of \"%s\" is unacceptable with the"
+                                            + " current URI range settings",
+                                    target.connectUrl));
+                }
+            } catch (MalformedURLException me) {
+                throw new IllegalArgumentException();
```

Comment:
Should the `MalformedURLException` be passed in as the cause of the `IllegalArgumentException`?